cardano-db-sync/schema

```diff
@@ -0,0 +1 @@
+../schema/
```

Comment:
Is that needed?

Reply:
This needs to stay so cabal knows about the path.